core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -67,21 +67,31 @@ angular.module('oppia').controller(
         }
       };
       $scope.canEditTranslation = false;
+      // The 'html' value is passed as an object as it is required for
+      // schema-based-editor. Otherwise the corrrectly updated value for
+      // the translation is not recieved from the editor when the translation
+      // is edited by the reviewer.
+      $scope.editedContent = {
+        html: $scope.translationHtml
+      };
+      $scope.errorMessage = '';
+      $scope.errorFound = false;
 
       $scope.updateSuggestion = function() {
-        const updatedTranslation = $scope.editedContent;
+        const updatedTranslation = $scope.editedContent.html;
         const suggestionId = $scope.activeSuggestion.suggestion_id;
+        $scope.tmpTranslationHtml = $scope.translationHtml;
```

Comment:
I think something like `preEditTranslationHtml` would be clearer to signify that this is the html corresponding to the original HTML.

Reply:
Done